model/schema/allele.yaml

```diff
@@ -235,25 +233,41 @@ classes:
       note_dto:
         required: true
 
-  AlleleSynonymSlotAnnotation:
-    is_a: SlotAnnotation
+  AlleleSymbolSlotAnnotation:
+    description: >-
+      The one current symbol for the allele.
+    is_a: NameSlotAnnotation
     slots:
       - single_allele
-      - synonym
-      - synonym_scope
     slot_usage:
       single_allele:
         required: true
-      synonym:
+      name_type:
+        any_of:
+          - equals_string: symbol
```

Comment:
I'm wondering: I know the name_type_name is a DTO class slot and so the range is "string" and therefore we can use the "equals_string" assertion, but I wonder if we can apply the same constraint for "name_type" where the range is actually 'VocabularyTerm'?

Reply:
I had the same thought, but I'm not sure these "rules" have much impact on anything (at least not anything we are using LinkML for) so I left them in as a potentially useful reminder for anyone looking at the model and not looking at the DTO classes.  Maybe better to add them as `notes` under `slot_usage` - I'll make that change

Reply:
Changed to note